<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLocate</class>
 <widget class="QDialog" name="DialogLocate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labRow">
       <property name="text">
        <string>行号</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labCol">
       <property name="text">
        <string>列号</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labSetText">
       <property name="text">
        <string>设定文字</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditText"/>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxColumn"/>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBoxRow"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolButton" name="btnClose">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnSetText">
     <property name="text">
      <string>设定文字</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogLocate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
